shanoir-ng-studies/src/main/java/org/shanoir/ng/study/model/Study.java

```diff
@@ -160,7 +160,7 @@ public class Study extends HalEntity {
 	@OneToMany(fetch = FetchType.LAZY, mappedBy = "study", cascade = CascadeType.ALL, orphanRemoval = true)
 	private List<Tag> tags;
 
-	private String publicDescription;
+	private String description;
```

Comment:
I think you'll need a SQL update to:
- Change the column name
- Update the column size